### PR TITLE
Add extraVolumeSecrets chart value

### PR DESCRIPTION
**Pull request by avorima**
This value can be used to render `volumes` and `volumeMounts`.

### CODE REVIEW

File: charts/cluster-autoscaler-chart/values.yaml [hunk 1]
```diff
@@ -150,6 +150,9 @@ extraArgs:
 # extraEnv -- Additional container environment variables.
 extraEnv: {}
 
+# extraVolumeSecrets -- Additional volumes to mount from Secrets.
```
Comment (gjtempleton) [inline charts/cluster-autoscaler-chart/values.yaml:153]: Can you include a full example of using this commented out given there's a number of sub-options here please.

**Comment by gjtempleton** [comment]
Hey, thanks for the PR!

One request for you from a first look over it.

**Comment by foriequal0** [comment]
I've found that my PR will conflict with this: https://github.com/kubernetes/autoscaler/pull/3725

**Comment by avorima** [comment]
@foriequal0 My intent is that extra volumes and volumeMounts can be added for any cloud provider, so I'd prefer not to move it into an `else if`-block

File: charts/cluster-autoscaler/Chart.yaml [hunk 2]
```diff
@@ -17,4 +17,4 @@ name: cluster-autoscaler
 sources:
   - https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler
 type: application
-version: 9.1.0
+version: 9.1.1
```
Comment (gjtempleton) [inline charts/cluster-autoscaler/Chart.yaml:20]: I'd prefer if we stuck to SemVer here and bumped the minor version as new functionality is being added

**Pull request merged**